src/test/java/org/graylog/integrations/dataadapters/GreyNoiseCommunityIpLookupAdapterTest.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog.integrations.dataadapters;
+
+import okhttp3.MediaType;
+import okhttp3.Protocol;
+import okhttp3.Request;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+import org.graylog2.plugin.lookup.LookupResult;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Map;
+
+public class GreyNoiseCommunityIpLookupAdapterTest {
+
+    private Request request;
+
+    @Before
+    public void setup() {
+
+        request = new Request.Builder()
+                .url(GreyNoiseCommunityIpLookupAdapter.GREYNOISE_ENDPOINT)
+                .build();
+    }
+
+    @Test public void testParseSuccess(){
+
+        String string = "{\"ip\": \"1.2.3.4\",\"noise\": false,\"riot\": true,\"classification\": \"benign\",\"name\": \"Cloudflare\",\"link\": \"https://viz.greynoise.io/riot/1.2.3.4\",\"last_seen\": \"2020-01-01\",\"message\": \"Success\"}";
```

Comment:
I get a slight feeling that the JSON is easier to maintain in a resource file, particularly since it scrolls to the right. But, then again, this is a simple flat object. There's no explicit project standard for this, so it's really on a case-by-basis. Whatever you and the others think is best here is fine with me.